frontend/src/components/my-parking-lot-reservations/parking-lots-filter.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import {
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  OutlinedInput,
+  Select,
+  SelectChangeEvent,
+  TextField
+} from '@mui/material';
+import { logger } from '../../logger';
+import { ParkingLotModel } from '../../models';
+import { MyParkingLotsFilterData } from '../../../pages/my-parking-lot';
+
+interface ParkingLotsFilterProps {
+  updateFilter: (filter: MyParkingLotsFilterData) => void;
+  parkingLots: ParkingLotModel[];
+}
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250
+    }
+  }
+};
+
+const ParkingLotsFilter = ({
+  updateFilter,
+  parkingLots
+}: ParkingLotsFilterProps) => {
+  const [searchTerm, setSearchTerm] = useState<string>('');
+
+  const [parkingLotName, setParkingLotName] = React.useState<string[]>([]);
+  const handleChange = (event: SelectChangeEvent<typeof parkingLotName>) => {
+    logger.log(event);
+    const {
+      target: { value }
+    } = event;
+    setParkingLotName(typeof value === 'string' ? value.split(',') : value);
+  };
+
+  useEffect(() => {
+    updateFilter({ names: new Set(parkingLotName), searchTerm: searchTerm });
+  }, [parkingLotName, searchTerm]);
+
+  return (
+    <Grid
+      container
+      justifyContent="space-between"
+      display="flex"
+      spacing={1}
+      rowSpacing={2}
+      padding={2}
+    >
+      <Grid item md={5} lg={5} xl={5} sm={8}>
+        <TextField
+          label={'Suchbegriff'}
+          fullWidth={true}
```

Comment:
```suggestion
          fullWidth
```
true state doesn't have to be defined